app/services/taxonomy/fetch_all_taxons.rb

```diff
@@ -1,5 +1,5 @@
 module Taxonomy
-  class TaxonFetcher
+  class FetchAllTaxons
```

Comment:
That makes sense to me. We could even move them into the existing `Taxon` model, and have make `all` a class method. If you don't think that's a good idea, then let's just move this class into `app/models` for now. 👍